sql/mob_droplist.sql

```diff
@@ -21724,9 +21717,9 @@ INSERT INTO `mob_droplist` VALUES (2677,0,0,1000,15138,@VRARE);  -- Valor Leggin
 INSERT INTO `mob_droplist` VALUES (2677,2,0,1000,1449,0);        -- Tukuku Whiteshell (Steal)
 
 -- ZoneID:  73 - Wyvern
-INSERT INTO `mob_droplist` VALUES (2678,0,0,1000,5366,@UNCOMMON); -- Castellanus Cell (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (2678,0,0,1000,5367,@UNCOMMON); -- Cumulus Cell (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (2678,0,0,1000,5369,@UNCOMMON); -- Stratus Cell (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (2678,0,0,1000,5367,@VCOMMON); -- Cumulus Cell (Vert Common, 24%)
```

Comment:
Typo

Reply:
Thanks, yea saw also but left lol, I'll be going back to add full comments later, but I'm sure some changes will be coming when I get into the mobs themselves and see how lsb works with eden, I most likely will have to port some of the treasure listeners over to accommodate

Reply:
got ya, editor found a random whitespace from before also